spec.bs

```diff
@@ -2137,6 +2139,35 @@ The output of running a Protected Audience `scoreAd()` script.
 
 </dl>
 
+This is returned via the following type:
+<pre class="idl">
+dictionary ScoreAdOutput {
+  required double desirability;
+  double bid;
+  boolean allowComponentAuction = false;
+};
+</pre>
+
+<div algorithm>
+
+To <dfn>process scoreAd output</dfn> given a |result|, a [=ECMAScript/Completion Record=]:
+  1. If |result| is an an [=abrupt completion=], return failure.
+  1. TODO: the code has special handling of where the return value is a straight up double, do we want to keep it? It is in the explainer.
+  1. Let |resultIDL| be the result of [=converted to an IDL value|converting=]
+          |result| to a {{ScoreAdOutput}}.
+  1. TODO: Shouldn't this be the completion's value that's being converted?

```

Comment:
yeah I think so.

Reply:
That's right. We can reference it via `result.[[Value]]`